src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/send/activity/ScheduleSendNotificationDateRetrieveActivity.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.workflow.wf.pagopa.send.activity;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+@ActivityInterface
+public interface ScheduleSendNotificationDateRetrieveActivity {
+  @ActivityMethod
+  void scheduleSendNotificationDateRetrieve(String sendNotificationId);
```

Comment:
on Confluence we have given to it a different name (https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1601241160/Notifica+SEND)
```suggestion
  void scheduleSendNotificationDateRetrieveWF(String sendNotificationId);
```